shared/src/pool_fetching.rs

```diff
@@ -183,6 +184,10 @@ impl BaselineSolvable for Pool {
             price
         })
     }
+
+    fn gas_cost(&self) -> usize {
+        POOL_SWAP_GAS_COST
```

Comment:
I assume this is a fixed cost for Uniswap-Like-Pools. Is this value substantially different for 1inch and will this need to be adjusted for Balancer?

Reply:
Yes, this likely has to be different for each BaselineSolvable (it's also only a rough estimate right now as the concrete swap costs depend on the concrete ERC20 token logic)